bus-mapping/src/circuit_input_builder.rs

```diff
@@ -442,19 +445,23 @@ impl CircuitInputBuilder<DynamicCParams> {
 /// Return all the keccak inputs used during the processing of the current
 /// block.
 pub fn keccak_inputs(block: &Block, code_db: &CodeDB) -> Result<Vec<Vec<u8>>, Error> {
-    let mut keccak_inputs = Vec::new();
+    let mut keccak_inputs: HashSet<Vec<u8>> = HashSet::new();
```

Comment:
How about also adding debug log when detecting duplicate input?  It make thing easy to trace and fix root cause if have.

Reply:
i think "duplicate input" is not something wrong.  in sha3 opcode, in sign verify, in mpt trie, it would be common to produce some duplicate input

Reply:
got it, thanks for clarify!